recipes/rhash/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "rhash" %}
+{% set version = "1.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rhash/RHash/archive/v{{ version }}.tar.gz
+  sha256: 1f6daa0c066c94d2575f0aace76f34e5800c51f59f4b30029ddcfa9799564f98
+  patches:
+    # osx port patch, not accepted upstream
+    # https://sourceforge.net/p/rhash/feature-requests/23/
+    - librhash-osx.patch  # [osx]
```

Comment:
Could we open a fresh upstream issue about this? Seems that issue is 3yrs old. Even if they don't like this patch, it would be nice if they propose an alternative that would also work.